### ISSUE TITLE
textField title color not set to default color when move to other textField

### ISSUE THREAD

**Issue by shaharukhs** [Skyscanner/SkyFloatingLabelTextField#161]
# Report

## What did you do?

I have some textfields with textfield class `SkyFloatingLabelTextField`
changed selected line color
changed selected text color
added placeholder text
added title text

but when I click any textfield and some text and move to other textfield title color won't change to default color and show selected title color only.
but in your project REDME.md example shows different and its not working as per

## What did you expect to happen?

ℹ Please replace this with what you expected to happen.

## What happened instead?

ℹ Please replace this with of what happened instead.

## Environment

ℹ Please fill in this section with information about your environment

Xcode Version:
SkyFloatingLabelTextField Version: SkyFloatingLabelTextField (3.2.0)
Deployment Target: iOS 10.3
Base SDK: XCode 8.3.2
Method of Integration: CocoaPods

## Project that demonstrates the issue

I have attached demo project please check attachment.

[demoFloatingtextField.zip](https://github.com/Skyscanner/SkyFloatingLabelTextField/files/1189944/demoFloatingtextField.zip)

**Comment by k0nserv** [comment]
Yeah this looks like a bug to me. Thanks for the excellent report 👍

**Comment by Mycose** [comment]
You can fix it by removing `|| isHighlighted` from `fileprivate func updateTitleColor()`

**Comment by shaharukhs** [comment]
Its work after removing` || isHighlighted ` for now. Thank you.

**Comment by k0nserv** [comment]
Happy to accept a PR if you already have a fix in mind

**Comment by vCrespoP** [comment]
It would be really nice to include this, as if we modify the code but then update the dependencies, this bug would be back again.

Thanks!

**Comment by joaovbalmeida** [comment]
Same thing happening here, i have a view with 4 textfields, if i start filling them by tapping on the first one and proceeding trough them by the keyboard next button, just the one that i tapped stays with the selection color, the rest turns back to the normal color. After analyzing, i realized that only the fields that you tap stays with the selection color after populated, if you populate a field by getting to it trough the next button, the selection color does not stick to it.

**Comment by joaovbalmeida** [comment]
Managed to fix by simply setting fields isHighlighted = true in viewdidload

`//Correct SkyFloating Issue
     emailField.isHighlighted = true
     nameField.isHighlighted = true
     passwordField.isHighlighted = true
     confirmPasswordField.isHighlighted = true`

**Comment by joaovbalmeida** [comment]
Managed to fix by setting highlighted on viewdidload

`textField.isHighlighted = true`

**Comment by k0nserv** [comment]
Hey I had a closer look at this and I can't reproduce this with the demo project attached. Tapping on the fields work as expected and the selected style is only on the that's the first responder. If you manually set `isHighlighted` you should manually set it to `false`, but for normal stuff this isn't required at all as the control will change when it becomes first responder/resigns first responder

**Comment by aashishtamsya** [comment]
Try on deployment target: 10.3.3, device: iPad

**Comment by sathamkhussain** [comment]
How to change the placeholder color?
